pyads/ads.py

```diff
@@ -671,6 +682,11 @@ def read(
 
         return None
 
+    def get_symbol(self, name: str) -> AdsSymbol:
+        """Get a single symbol"""
```

Comment:
```suggestion
        """Get a single symbol by name."""
```
Please add a period at the end of the docstring.